packages/mdc-checkbox/_mixins.scss

```diff
@@ -329,6 +343,15 @@ $mdc-checkbox-ripple-target: ".mdc-checkbox__ripple";
   }
 }
 
+@mixin mdc-checkbox-touch-target($size: $mdc-checkbox-ripple-size, $query: mdc-feature-all()) {
```

Comment:
Update test/scss/_feature-targeting-test.scss test file with all the new mixins.

Reply:
👍Glad we have Sass tests! What do those tests do?

Reply:
This'll check if there are any leakage when $query is set to none.